src/android/CHIPTool/README.md

```diff
@@ -9,7 +9,7 @@ a CHIP device
 
 # Steps to build CHIP for Android apps
 
-Pre-conditions: Have Android SDK & NDK downloaded to your machine. Set the
+Pre-conditions: Have Android SDK 21 & NDK 21.0.6113669 downloaded to your machine. Set the
```

Comment:
Since there will be newer revisions of the Android NDK, we should not specify a particular version value.
The instructions here, correctly, do state that 'make sure your $ANDROID_NDK_HOME environment variable is pointing to the downloaded NDK package'.

Having the Android SDK version is fine.

Reply:
Thank you @vidhis88, but when I tried newer versos of NDK, the build failed with below error message `No version of NDK matched the requested version 21.0.6113669. Versions available locally: 21.3.6528147`

Reply:
I understand. But the same error, with a new version number will be seen if a newer version of NDK is released and someone follows this README to download version 21.0.6113669.
All I'm saying is that the versions are constantly updated by Android and so we should ensure that the downloaded NDK version matches the one in the environment variable, it doesn't matter which version.

Reply:
make sense, I remove NDK version from readme.